apis/nucleus/src/components/listbox/interactions/listbox-get-action-props.js

```diff
@@ -0,0 +1,28 @@
+export default ({ isPopover, showToolbar, containerRef, isLocked, listboxSelectionToolbarItems, selections }) => ({
```

Comment:
Named default export?
```suggestion
export default function getListboxActionProps({ isPopover, showToolbar, containerRef, isLocked, listboxSelectionToolbarItems, selections }) {
```